samples/build.gradle

```diff
@@ -19,7 +19,8 @@ android {
 }
 
 dependencies {
-  compile 'com.dlazaro66.qrcodereaderview:qrcodereaderview:2.0.0'
+  //compile 'com.dlazaro66.qrcodereaderview:qrcodereaderview:2.0.0'
+  compile project (":qrcodereaderview")
```

Comment:
I don't think this is the right way to go. The sample project goal is to show how the library should be used. Using a ```compile project``` dependency is not something that library consumers will be able to do. Also using the real binary that is being deployed to maven will protect the sample project from unexpected errors.